apps/staking/src/api.ts

```diff
@@ -210,12 +195,12 @@ export const loadData = async (context: Context): Promise<Data> => {
     walletAmount: ownerAtaAccount.amount,
     integrityStakingPublishers: publishers.map(({ pubkey: publisher }) => ({
       apyHistory: [], // TODO
-      isSelf: false, // TODO
+      isSelf: publisher === client.wallet.publicKey,
```

Comment:
fyi @keyvankhademi

Reply:
This is actually incorrect, a publisher stake account is not necessarily the owner of the stake account. I will create a function in the sdk to retrieve the publishers stake accounts.

Reply:
ah, ok, thanks -- I'll undo this line then